k8s/helm-charts/seldon-core-v2-servers/templates/seldon-v2-servers.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
   name: mlserver
   namespace: '{{ .Release.Namespace }}'
 spec:
+  maxReplicas: {{ .Values.mlserver.maxReplicas }}
```

Comment:
I think those should be updated here: `https://github.com/SeldonIO/seldon-core/blob/v2/k8s/kustomize/helm-servers/patch_mlserver.yaml` (and similarly for triton) in order for the changes not to be overwritten when running `make create-helm-charts` in the k8s dir.